src/Ledger/RolloverLedger/RolloverLedgerContainer.js

```diff
@@ -70,6 +72,8 @@ export const RolloverLedgerContainer = ({ resources, mutator, match, history, lo
   const { toFiscalYearId, toFiscalYearSeries } = location.state || {};
   const series = currentFiscalYear?.series;
 
+  const { fiscalYears } = useRolloverFiscalYears(series);
```

Comment:
In my opinion, the `useFiscalYearsBatch` hook is more suitable here, to which you can pass specific IDs of fiscal years that participate in the current rollover, smth like:
```
const { fiscalYears } = useFiscalYearsBatch([toFiscalYearId, fromFiscalYearId]);
```

Reply:
@usavkov-epam It is an exciting code and I don't want to create a regression. I have just moved the code above to prevent the "`fiscalYears`  was used before it was defined" issue